registry.json

```diff
@@ -3529,5 +3529,19 @@
     "telegram": "woltere",
     "operatorName": "Wolter",
     "github": "RV64-Stake-Pool"
+  },
+   {
+       "ticker": "KAKIS",                                                       
+       "poolId": "09ae473a0781eefa6345e2236a8ffa92c26dc151c93a8f67ae735860",  
+       "social": {              
+         "website":"https:/ada.stakingkakis.com/", 
+         "twitter": "",
+         "facebook": "",  
+         "instagram": "",
+         "youtube": ""
+       },
+       "telegram": "stakingkakis",                                             
```

Comment:
Please can you change this to the TG handle of a user and not a channel or group

Reply:
Done, please update accordingly